crates/core/src/host/wasm_common/module_host_actor.rs

```diff
@@ -604,23 +598,13 @@ impl<T: WasmInstance> WasmModuleInstance<T> {
                 }
             }
             Ok(Err(errmsg)) => {
-                stdb.rollback_mut_tx(&ctx, tx);
-
                 log::info!("reducer returned error: {errmsg}");
 
                 EventStatus::Failed(errmsg.into())
             }
-            Ok(Ok(())) => {
-                if let Some(tx_data) = stdb.commit_tx(&ctx, tx).unwrap() {
-                    EventStatus::Committed(DatabaseUpdate::from_writes(&tx_data))
-                } else {
-                    todo!("Write skew, you need to implement retries my man, T-dawg.");
-                }
-            }
+            Ok(Ok(())) => EventStatus::Committed(DatabaseUpdate::default()),
```

Comment:
nit: As commit is not happening here anymore, may be we can choose better name for`EventStatus::Committed` -> `EventStatus::YetToCommitted` or `EventStatus::ReducerSuccess`